artifacts-abcdef-rest_task_variant_task_one/robots.txt

```diff
@@ -0,0 +1,276 @@
+User-agent: *
```

Comment:
What is this file for? There are a couple more like this

Reply:
I believe they're detritus from the smoke test.